tests/8_breeding.js

```diff
@@ -163,6 +164,8 @@ module.exports = async function(contracts, [node], accounts, wallet) {
 
   console.log('sending breeding condition');
   await node.sendTx(condTx);
+  const rsp = await node.provider.send('eth_getTransactionReceipt', [condTx.hash()]);
+  assert(rsp.logs && rsp.logs.length > 0, 'no events emitted');
```

Comment:
don't we want to check the actual log message here?

Reply:
i don't know how to parse it 😬